nvflare/app_common/abstract/learner2.py

```diff
@@ -0,0 +1,240 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any, Union
+
+from nvflare.apis.dxo import DXO
+from nvflare.apis.fl_component import FLComponent
+from nvflare.apis.fl_context import FLContext
+
+
+class Learner2(FLComponent):
```

Comment:
Following the `LearnerExecutor` inside learner2_executor.py file,
Should we change this name to just `Learner`?

Reply:
Good point!

Reply:
Changed class name to Learner.